libretro/libretro.cpp

```diff
@@ -386,7 +399,7 @@ static void check_variables(CoreParameter &coreParam)
    g_Config.sLanguageIni = map_psp_language_to_i18n_locale(g_Config.iLanguage);
    i18nrepo.LoadIni(g_Config.sLanguageIni);
 
-   if (!PSP_IsInited() && ppsspp_internal_resolution.Update(&g_Config.iInternalResolution))
+   if (ppsspp_internal_resolution.Update(&g_Config.iInternalResolution) && !PSP_IsInited())
```

Comment:
Hm, does it not work when the game was already started?  Normally, PPSSPP can handle resizing at runtime.

-[Unknown]

Reply:
There are issues with the gl driver crashing or black screening when changed at runtime. So its been gated off for a while. It's on my list of things to try fix.